src/CMakeLists.txt

```diff
@@ -69,6 +69,28 @@ file (GLOB xournalpp_SOURCES_RECURSE
   pdf/popplerapi/*.cpp
 )
 
+if (WIN32)
```

Comment:
This code for generating the icons could probably be combined into the one below.

Reply:
I addressed this.